tests/Mono.Android-Tests/System.Net/NetworkInterfaces.cs

```diff
@@ -85,6 +85,9 @@ bool AddressesAreEqual (List <IPAddress> one, List <IPAddress> two)
 
 			bool HardwareAddressesAreEqual (byte[] one, byte[] two)
 			{
+				// Under API 33 .Net doesn't return the hardware address. So we need to ignore it
+				if (Android.OS.Build.VERSION.SdkInt == Android.OS.BuildVersionCodes.Tiramisu)
```

Comment:
@dellis1972: Shouldn't this use `>=` so that API-34 hits this codepath?  Or do we *want* it to fail on future API-34?

Reply:
So we could. but i'd rather change that in API 34 if or when we hit that issue.  This is only in a unit test not the actual System.Net code.